### ISSUE TITLE
HOw to install when your linux distro is arch?

### ISSUE THREAD

**Issue by Koraprognan** [v3n0m-Scanner/V3n0M-Scanner#205]
help please..

**Comment by vittring** [comment]
Install Python, clone this repo, install Venom.

It is included by default in [Black Arch](https://github.com/BlackArch/blackarch-iso/blob/master/full-iso/packages.x86_64). You can install it simply using pip after you git clone this repo or just use a Docker image, the choice is up to you and your particular skill set with Linux. Should make things easier for even the less tech savvy users.

To be blunt, refrain from such vague questions in the future or at least provide an attempt to fix the issue yourself.

**Comment by kattstof** [comment]
I would even dumb it down further and say download this repo, and just python v3n0m for added simplicity.
